badger/badger.go

```diff
@@ -20,7 +20,8 @@ func New(config ...Config) *Storage {
 	cfg := configDefault(config...)
 
 	// Set options
-	opt := badger.DefaultOptions(cfg.Database).WithTruncate(true).WithLogger(nil)
+	opt := cfg.BadgerOptions
+	// TODO see if as a new function similar to WithTruncate(true)
```

Comment:
please check this TODO

Reply:
Looks like this option or any similar no more exist

The most similar function that I found is [func (Options) WithCompactL0OnClose](https://pkg.go.dev/github.com/dgraph-io/badger/v3@v3.2011.1#Options.WithCompactL0OnClose)

So, I'll remove the TODO